### PR TITLE
Make /api/eda and /api/eda/v1 endpoints available via API service

### PR DESCRIPTION
**Pull request by rooftopcellist**
@Alex-Izquierdo this is not yet working, could I get your feedback on where I've gone wrong?

### CODE REVIEW

**Review by Alex-Izquierdo — APPROVED** [review]
I think it is fine as it is.

**Comment by Alex-Izquierdo** [comment]
As we discussed privately, the nginx conf of eda-ui needs to be changed as well.

File: roles/eda/templates/eda.configmap.yaml.j2 [hunk 1]
```diff
@@ -75,21 +77,19 @@ data:
...
                 proxy_set_header Origin http://{{ api_server_name }}:{{ api_nginx_port }};
                 proxy_set_header Host $http_host;
-                proxy_set_header X-Forwarded-Host $host;
                 proxy_set_header X-Forwarded-Proto $remote_scheme;
                 proxy_set_header X-Forwarded-Port $server_port;
                 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             }
 
-            location ~ ^/static/ {
+            location ~ ^/api/eda/static/ {
```
Comment (Alex-Izquierdo) [inline roles/eda/templates/eda.configmap.yaml.j2:89]: you don't need this location, is mostly duplicated with the previous one. `location ~ ^/api/eda`

**Comment by rooftopcellist** [comment]
Updated.  Also, now redirects work with curls for /api/eda when using the api service directly.

```
sh-5.1$ curl eda-api:8000/api/eda -k -L -u admin:password
{"current_version":"http://eda-api:8000/api/eda/v1/","available_versions":{"v1":"http://eda-api:8000/api/eda/v1/"}}
```

```
sh-5.1$ curl eda-ui:80/api/eda -k -L -u admin:password
{"current_version":"http://eda-ui/api/eda/v1/","available_versions":{"v1":"http://eda-ui/api/eda/v1/"}}
```

**Pull request merged**